metagpt/actions/action.py

```diff
@@ -63,24 +68,21 @@ async def _aask_v1(
             system_msgs = []
         system_msgs.append(self.prefix)
         content = await self.llm.aask(prompt, system_msgs)
-        logger.debug(content)
+        logger.debug(f"llm raw output:\n{content}")
         output_class = ActionOutput.create_model_class(output_class_name, output_data_mapping)
+        output_class_fields = list(output_class.schema()["properties"].keys())  # Custom ActionOutput's fields
 
         if format == "json":
```

Comment:
do we need to use it when llm is GPT-4?

Reply:
gpt-4 allways gets right instruction result, REPAIR_LLM_OUTPUT can be False